rules/use-the-best-email-templates/rule.md

```diff
@@ -16,7 +16,9 @@ When sending repetitive emails, consistency across the organization is crucial.
 
 <!--endintro-->
 
-### Outlook Email Templates
+### Outlook Templates

```

Comment:
Hi @Tanya-Leahy 

- [ ] 1. Please make this and all the other headings `<h2> ` instead of 3

Reply:
Done - all headings are now h2.